src/AlgebraOfGraphics.jl

```diff
@@ -37,6 +37,9 @@ export draw, draw!
 export facet!, colorbar!, legend!
 export set_aog_theme!
 
+# Minor piracy to fix Makie issues
+Makie.MakieLayout.convert_for_attribute(t::Type{Makie.FreeTypeAbstraction.FTFont}, x::Makie.FreeTypeAbstraction.FTFont) = to_font(x)
```

Comment:
Any intuition as to why this is needed? It may be best to upstream this in Makie anyway.

Reply:
no - probably @SimonDanisch or @jkrumbiegel can help?

Reply:
ah this is just because FTFont is one of the only things I defined for conversion in the Block refactor, fields of Axis, Axis3 etc can now be of fixed types. This thing is probably because you're passing an already converted font and there's an ambiguity error?